access/src/main/webapp/WEB-INF/jsp/fullRecord/neighborList.jsp

```diff
@@ -29,8 +29,18 @@
 			<c:set var="currentItemClass" scope="page">
 				<c:if test="${briefObject.id == neighbor.id}"> current_item</c:if>
 			</c:set>
+			<c:set var="markedForDeletion"  value="${cdr:contains(neighbor.status, 'Marked For Deletion')}" scope="request"/>
+			<c:choose>
+				<c:when test="${markedForDeletion}">
+					<c:set var="isDeleted" value="deleted" scope="request"/>
```

Comment:
this is overriding the other `isDeleted` value, I'd suggest using a different variable name, like `neighborIsDeleted` so we don't end up with unintended side effects. Also you can use the page scope for this variable.